deployment/helm/README.md

```diff
@@ -0,0 +1,166 @@
+
+## Tips
+
+- This project is based on [docker-compose-no-secty.yml](https://github.com/edgexfoundry/edgex-compose/blob/jakarta/docker-compose-no-secty.yml),
+you can implement your customized version based on this.
+- Since the EdgeX pods communicates with each other through the kubernetes service name, make sure the kubernetes DNS is enabled.
+- Since other edgex services need to rely on consul to obtain configuration or register themselves to consul, other services cannot run normally until consul starts successfully.
+- Unlike the docker-compose files for this release (which use a separate Docker volume container), the manifest files mount host based volumes as follows:
+
+1、edgex-core-consul's /consul/config directory is mapped to the host's /mnt/edgex-consul-config directory.
+
+2、edgex-core-consul's /consul/data directory is mapped to the host's /mnt/edgex-consul-data directory.
+
+3、edgex-db's /data/db directory is mapped to the host's /mnt/edgex-db directory.
+
+4、edgex-kuiper's /kuiper/data directory is mapped to the host's /mnt/edgex-kuiper-data directory.
+
+- NodePort is enabled by default. According to default NodePort range(30000～32767), EdgeX NodePort mappings are as follows. 
+
+| EdgeX Service Name          | Exposed Port 
+| :-------------------------- | ------------- 
+| edgex-core-data             | 59880         
+| edgex-core-metadata         | 59881         
+| edgex-core-command          | 59882         
+| edgex-support-notifications | 59860         
+| edgex-support-scheduler     | 59861         
+| edgex-app-rules-engine      | 59701         
+| edgex-kuiper                | 59720         
+| edgex-device-rest           | 59986         
+| edgex-device-virtual        | 59900         
+| edgex-ui                    | 4000          
+| edgex-sys-mgmt-agent        | 58890         
+| edgex-redis                 | Not Exposed 
+| edgex-core-consul           | 8500          
+
+## Enabling security features
+
+The helm chart uses an Kubernetes ingress controller in lieu of a Kong API gateway.
+The ingress routes are configured to require client-side TLS authentication,
+which replaces the Kong JWT authentication method.
+
+### Prerequisites
+
+Before starting, make sure you have curl and openssl installed locally.
+These tools are need to generate TLS assets and test the configuration.
+
+These instructions also assume that your Kubernetes cluster has an installed
+ingress controller and an installed load balancer.
+The examples were built with a stock Ngnix ingress controller,
+and a MetalLB load balancer (with an assigned IP of 192.168.122.200).
+Adaptations may be needed if your cluster uses something different.
+
+To install the Nginx ingress controller,
+follow the instructions at
+<https://kubernetes.github.io/ingress-nginx/deploy/#quick-start>
+
+To install MetalLB,
+follow the instructions at
+<https://metallb.universe.tf/installation/#installation-with-helm>
```

Comment:
Do we have recommendations with the sample to use L2 or BGP mode for MetallB?

Reply:
No recommendation offered.  This is merely documentation as to what component was used to provide the IP.